src/tokens/Assets.tsx

```diff
@@ -205,7 +205,7 @@ function AssetsScreen({ navigation, route }: Props) {
   }
 
   return (
-    <>
+    <Animated.View>
```

Comment:
A weird edge case case were an animated view wrapped in a fragment can cause transparency issues.